Gemfile.lock

```diff
@@ -10,16 +10,16 @@ GIT
 PATH
   remote: .
   specs:
-    cassandra-utils (0.2.4)
-      daemon_runner (~> 0.2)
+    cassandra-utils (0.0.1)
```

Comment:
@rdickey-r7, we're using [thor-scmversion](https://github.com/RiotGamesMinions/thor-scmversion) to handle versioning in the CI job. It reads the version from tags in the Git repo, so the version in Gemfile.lock is ignored.